File: ABC/166-2020-05-03/B.c
#include<stdio.h>

int main(){
    int N, K;

    scanf("%d %d", &N, &K);
    //printf("N:%d,K:%d\n", N, K);//

    int d[K];
    int A[K][N];
    int sunuke[N];
    int trick = 0;

    int i,j;

    for (i = 0; i < N;i++){
        sunuke[i] = 0;
    }

        for (i = 0; i < K; i++)
        {
            scanf("%d", &d[i]);
            //printf("d:%d\n", d[i]); //
            for (j = 0; j < d[i]; j++)
            {
                scanf("%d", &A[i][j]);
                //printf("A:%d\n", A[i][j]);
                sunuke[A[i][j] - 1] += 1;
                //printf("j:%d\n", j);
            }
        }

    for (i = 0; i < N;i++){
        //printf("sunuke:%d\n", sunuke[i]);
        if(sunuke[i] == 0){
            trick++;
        }
    }

    printf("%d", trick);

    return 0;
}

File: ABC/166-2020-05-03/B.py
N, K = map(int, input().split())

d = []
A = [[]]
i = 0
j = 0
k = 0
trick = 0
sudeni = [N - 1]
sunuke = []

for i in range(K):
    d[i] = int(input())
    for j in range(d[i] - 1):
        A[i][j] = int(input())
        sudeni[j] = A[i][j]
    for k in range(len(sudeni) - 1):
        sunuke[sudeni[k]] += 1

print(sunuke)